platform/android/java/lib/src/org/godotengine/godot/GodotIO.java

```diff
@@ -338,10 +339,25 @@ public void dir_close(int id) {
 		dirs.remove(id);
 	}
 
+	/////////////////////////
```

Comment:
Instead of routing through GodotIO, can you instead follow a pattern similar to what we're doing for GodotIO access  in the native code:

- Make `GodotNetUtils netUtils` a static variable in `Godot.java` and initialize there.
- In the `java_godot_lib_jni.cpp` use `godot_java->get_member_object` to get a `jobject` for `GodotNetUtils`.
- Pass that `jobject` to `NetSocketAndroid` directly and use that to setup the `multicastLock*` methods.